assemble/Multiphase.F90

```diff
@@ -366,14 +369,18 @@ subroutine assemble_fluid_particle_drag(istate_fluid, istate_particle)
 
                type(scalar_field), pointer :: vfrac_fluid, vfrac_particle
                type(scalar_field), pointer :: density_fluid, density_particle
+               type(scalar_field), pointer :: pd_scalar_field ! scalar field defining particle diameter 
```

Comment:
This needs a few changes for consistency, since you use a combination of `pd`, `dia` and `d` now. I'd prefer this to be kept as `d` throughout (i.e. `d_field`, `d_cap_lower`, etc).